tests/helpers/test_device_registry.py

```diff
@@ -3210,20 +3210,31 @@ async def test_update_remove_config_subentries(
     }
 
 
+@pytest.mark.parametrize(
+    ("initial_area", "device_area_id", "number_of_areas"),
+    [
+        (None, None, 0),
+        ("Living Room", "living_room", 1),
+    ],
+)
 async def test_update_suggested_area(
     hass: HomeAssistant,
     device_registry: dr.DeviceRegistry,
     area_registry: ar.AreaRegistry,
     mock_config_entry: MockConfigEntry,
+    initial_area: str | None,
+    device_area_id: str | None,
+    number_of_areas: int,
 ) -> None:
     """Verify that we can update the suggested area version of a device."""
```

Comment:
```suggestion
    """Verify that we can't update the suggested area version of a device."""
```
also lets update the test name to `test_update_suggested_area_ignored` or something, since it is not testing the update anymore

Reply:
Yes, that still happens here:
https://github.com/home-assistant/core/blob/c6917b9dc6ca567349cf9cd60b9dcf5bc7638a53/homeassistant/helpers/device_registry.py#L1224-L1225

Reply:
Oh right. :+1: